code/lib/manager-api/src/modules/channel.ts

```diff
@@ -8,8 +8,10 @@ import type { API, ModuleFn } from '../index';
 export interface SubAPI {
   getChannel: () => API_Provider<API>['channel'];
```

Comment:
Should we have `getServerChannel` also? What about equivalents of `off`, `once`, etc?

I find the names `onServerAction` and `emitServerAction` kind of awkward. What about adding an extra argument to the other functions `on('server', 'event', ...)` or similar? Is that too confusing?

Reply:
@tmeasday I propose we do this, then later merge the 2 channels, and when we do that completely drop `onServerAction` and `emitServerAction` entirely.

So in the future `.on` will simply listen to both, and `.emit` will emit to both.

I don't want to do that today, because it would complicate/delay.